operations/kubernetes/Manage_Sealed_Secrets.md

```diff
@@ -0,0 +1,181 @@
+      ncn-m001# kubectl -n kube-system get secret sealed-secrets-key -o jsonpath='{.data.tls\.key}' | base64 -d - > certs/sealed_secrets.key
+      ```
+
+1. (Optional) Prevent tracked sealed secrets from being regenerated.
+    
+    1. Remove the sealed secrets not being regenerated from the `spec.kubernetes.tracked_sealed_secrets` list in `/root/site-init/${CSM_DISTDIR}/shasta-cfg/customizations.yaml` prior to executing the remaining steps in this section.
+
+    2. Retain the REDS/MEDS/RTS credentials.
+
+       ```bash
+       linux# yq delete -i ./${CSM_DISTDIR}/shasta-cfg/customizations.yaml spec.kubernetes.tracked_sealed_secrets.cray_reds_credentials
+       linux# yq delete -i ./${CSM_DISTDIR}/shasta-cfg/customizations.yaml spec.kubernetes.tracked_sealed_secrets.cray_meds_credentials
+       linux# yq delete -i ./${CSM_DISTDIR}/shasta-cfg/customizations.yaml spec.kubernetes.tracked_sealed_secrets.cray_hms_rts_credentials
+       ```
+
+2. Prepare to generate sealed secrets.
+   
+   ```bash
+   ncn-m001# ./utils/secrets-reencrypt.sh customizations.yaml ./certs/sealed_secrets.key ./certs/sealed_secrets.crt
+   ```
+      
+3. Encrypt the static values in the customizations.yaml file after making changes.
+
+   The following command must be run within the site-init directory.
+
+   ```bash
+   ncn-m001# ./utils/secrets-seed-customizations.sh customizations.yaml
+   ```
+
+   Expected output looks similar to:
+
+      ```bash
+      Creating Sealed Secret keycloak-certs
+      Generating type static_b64...
+      Creating Sealed Secret keycloak-master-admin-auth
+      Generating type static...
+      Generating type static...
+      Generating type randstr...
+      Generating type static...
+      Creating Sealed Secret cray_reds_credentials
+      Generating type static...
+      Generating type static...
+      Creating Sealed Secret cray_meds_credentials
+      Generating type static...
+      Creating Sealed Secret cray_hms_rts_credentials
+      Generating type static...
+      Generating type static...
+      Creating Sealed Secret vcs-user-credentials
+      Generating type randstr...
+      Generating type static...
+      Creating Sealed Secret generated-platform-ca-1
+      Generating type platform_ca...
+      Creating Sealed Secret pals-config
+      Generating type zmq_curve...
+      Generating type zmq_curve...
+      Creating Sealed Secret munge-secret
+      Generating type randstr...
+      Creating Sealed Secret slurmdb-secret
+      Generating type static...
+      Generating type static...
+      Generating type randstr...
+      Generating type randstr...
+      Creating Sealed Secret keycloak-users-localize
+      Generating type static...
+      ```
+
+
+
+### Prevent Regeneration of Tracked Sealed Secrets
+
+In order to prevent tracked sealed secrets from being regenerated, they 
```

Comment:
This uses the term "tracked sealed secrets" before it's been defined what a tracked sealed secret is.